Types/TransactionAttachment.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
This specifically belongs to a transaction, so I would store it with the transaction domain instead of as a global type